src/eea/themecentre/browser/datareporting.py

```diff
@@ -62,12 +59,22 @@ def relatedReportingObligations(self):
             reso = res.getObject()
             for rodid in reso.reportingObligations:
                 if rodid not in rodsdone:
+
+                    try:
+                        rodinfo = rodsinfo[int(rodid)]
+                    except KeyError, err:
+                        # Obligation not found
+                        logger.exception(err)
```

Comment:
As this is a known exception, as stated by the comment, maybe we should use `logger.warn` or a higher logging level in order not to get this reported again by sentry/graylog.

Reply:
fixed